README.md

```diff
@@ -8,6 +8,16 @@
 - [How To Build](https://wiki.status.im/Building_status-go)
 - [Notes on Bindings](https://wiki.status.im/Status-go_Binding_notes)
 
+# Notes on Git/GitHub workflow
+
+1. Squash PR before merging.
+   You can do it either with GitHub api or locally if you want to preserve signature.
+   Don't use `Create merge commit` option on GitHub. It pollutes history with meaningless information.
```

Comment:
Actually, this part is unneeded as Merge button functionality can be restricted via repository configuration. We can restrict any of the available options.

Reply:
@divan can you do that now? that's a nice thing, I don't see us ever wanting to have merge commits in the tree. I don't seem to have rights to tweak these settings now.